swaybar/render.c

```diff
@@ -172,7 +172,7 @@ static void render_block(struct window *window, struct config *config, struct st
 
 }
 
-static char *handle_workspace_number(bool strip_num, const char *ws_name) {
+static const char *strip_workspace_name(bool strip_num, const char *ws_name) {
```

Comment:
Don't need for the in strip_num argument since all call sites pass in swaybar.config->strip_workspace_numbers which is available here.